runtime/src/accounts.rs

```diff
@@ -785,21 +786,51 @@ impl Accounts {
         bank_id: BankId,
         index_key: &IndexKey,
         filter: F,
-        config: ScanConfig,
+        config: &ScanConfig,
+        byte_limit: Option<usize>,
     ) -> ScanResult<Vec<(Pubkey, AccountSharedData)>> {
-        self.accounts_db
+        let sum = AtomicUsize::default();
+        let config = ScanConfig {
+            abort: Some(config.abort.as_ref().map(Arc::clone).unwrap_or_default()),
```

Comment:
```suggestion
            abort: config.abort.as_ref().map(Arc::clone),
```

Reply:
I always want 'abort' to be Some if it was None. If it was Some, I want to keep that Arc. Maybe there is another method that does what I want here.